build-constraints.yaml

```diff
@@ -1620,27 +1620,6 @@ packages:
         - netcode-io
         - reliable-io
 
-    "Emanuel Borsboom <manny@fpcomplete.com> @borsboom":
-        - BoundedChan
-        - broadcast-chan
-        - fuzzcheck
-        - here
-        - hlibgit2
-        - gitlib-libgit2 < 0 # 3.1.2.1 compile fail
-        - interpolatedstring-perl6
-        - iproute
-        - missing-foreign
-        - multimap
-        - parallel-io
-        - text-binary
-        - Chart-cairo
-        - ghc-events
-        - monad-extras
-        - optparse-simple
-        - hpack
-        - bindings-uname
-        - stack < 9.9.9 || > 9.9.9
```

Comment:
@mpilgrem Does this need to get assigned to you?

Reply:
I am happy for `stack` to be assigned to me - although I have questions over its suitability as a Stackage snapshot package; see https://github.com/commercialhaskell/stack/issues/6159#issuecomment-1596116151

In respect of `hpack`, I think @sol is the person to ask.

The others, I have had no involvement with the packages and `stack` already takes up most of my free time.